### PR TITLE
[docs](load) split insert into values and insert into select

### PR DESCRIPTION
**Pull request by kaijchen**
# Versions 

- [x] dev
- [x] 3.0
- [x] 2.1
- [ ] 2.0

# Languages

- [x] Chinese
- [x] English

### CODE REVIEW

**Comment by KassieZ** [comment]
please also update 2.1/3.0 as well.

**Comment by KassieZ** [comment]
request review @wangtianyi2004

**Comment by wangtianyi2004** [comment]
1. 快速上手修改为 操作示例
2. 参考手册部分移除

**Comment by wangtianyi2004** [comment]
LGTM

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-3.0/data-operate/import/import-way/insert-into-values-manual.md [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+
+## 基本原理
+
+在使用 INSERT INTO VALUES 时，需要通过 MySQL 协议发起导入作业给 FE 节点，FE 会生成执行计划，执行计划中前部是查询相关的算子，最后一个是 OlapTableSink 算子，用于将查询结果写到目标表中。执行计划会被发送给 BE 节点执行，Doris 会选定一个节点做为 Coordinator 节点，Coordinator 节点负责接受数据并分发数据到其他节点上。
+
+## 快速上手
+
+INSERT INTO VALUES 通过 MySQL 协议提交和传输。下例以 MySQL 命令行为例，演示通过 INSERT INTO VALUES 提交导入作业。
+
+详细语法可以参见 [INSERT INTO](../../../sql-manual/sql-statements/Data-Manipulation-Statements/Manipulation/INSERT.md)。
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-3.0/data-operate/import/import-way/insert-into-values-manual.md:45]: New path: ../../../sql-manual/sql-statements/data-modification/DML/INSERT.md

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-3.0/data-operate/import/import-way/insert-into-values-manual.md [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+1 row in set (0.00 sec)
+```
+
+常见的错误的原因有：源数据列长度超过目的数据列长度、列类型不匹配、分区不匹配、列顺序不匹配等。
+
+可以通过环境变量 `enable_insert_strict`来控制 INSERT INTO 是否忽略错误行。
+
+## 更多帮助
+
+关于 Insert Into 使用的更多详细语法，请参阅 [INSERT INTO](../../../sql-manual/sql-statements/Data-Manipulation-Statements/Manipulation/INSERT) 命令手册，也可以在 MySQL 客户端命令行下输入 `HELP INSERT` 获取更多帮助信息。
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-3.0/data-operate/import/import-way/insert-into-values-manual.md:309]: New path: ../../../sql-manual/sql-statements/data-modification/DML/INSERT.md

File: docs/data-operate/import/import-way/insert-into-values-manual.md [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+
+```sql
+mysql> INSERT INTO test_table (user_id, name, age) VALUES (1, "Emily", 25), (2, "Benjamin", 35), (3, "Olivia", 28), (NULL, "Alexander", 60), (5, "Ava", 17);
+Query OK, 4 rows affected, 1 warning (0.04 sec)
+{'label':'label_a8d99ae931194d2b_93357aac59981a18', 'status':'VISIBLE', 'txnId':'61068'}
+```
+
+`Query OK` indicates successful execution. `4 rows affected` indicates that a total of 4 rows of data have been imported. `1 warnings` indicates the number of rows that were filtered out. 
+
+You can use the [SHOW LOAD](../../../sql-manual/sql-statements/Show-Statements/SHOW-LOAD/) statement to view the filtered rows.
```
Comment (KassieZ) [inline docs/data-operate/import/import-way/insert-into-values-manual.md:199]: new path : ../../../sql-statements/data-modification/load-and-export/SHOW-LOAD.md

File: docs/data-operate/import/import-way/insert-into-values-manual.md [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+
+```sql
+mysql> INSERT INTO test_table (user_id, name, age) VALUES (1, "Emily", 25), (2, "Benjamin", 35), (3, "Olivia", 28), (4, "Alexander", 60), (5, "Ava", 17);
+Query OK, 5 rows affected (0.04 sec)
+{'label':'label_78bf5396d9594d4d_a8d9a914af40f73d', 'status':'COMMITTED', 'txnId':'61074'}
+```
+
+The invisible state of data is temporary, and the data will eventually become visible. 
+
+You can check the visibility status of a batch of data using the [SHOW TRANSACTION](../../../sql-manual/sql-statements/Show-Statements/SHOW-TRANSACTION/) statement.
```
Comment (KassieZ) [inline docs/data-operate/import/import-way/insert-into-values-manual.md:238]: new path : ../../../sql-manual/sql-statements/transaction/SHOW-TRANSACTION.md

File: docs/data-operate/import/import-way/insert-into-values-manual.md [hunk 5]
```diff
@@ -0,0 +1,305 @@
...
+1 row in set (0.00 sec)
+```
+
+Common reasons for errors include: source data column length exceeding destination column length, column type mismatch, partition mismatch, and column order mismatch.
+
+You can control whether INSERT INTO ignores error rows by configuring the environment variable `enable_insert_strict`.
+
+## More help
+
+For more detailed syntax on INSERT INTO, refer to the [INSERT INTO](../../../sql-manual/sql-statements/Data-Manipulation-Statements/Manipulation/INSERT/) command manual. You can also type `HELP INSERT` at the MySQL client command line for further information.
```
Comment (KassieZ) [inline docs/data-operate/import/import-way/insert-into-values-manual.md:305]: New path : ../../../sql-statements/data-modification/DML/INSERT.md

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data-operate/import/import-way/insert-into-values-manual.md [hunk 6]
```diff
@@ -0,0 +1,309 @@
...
+
+## 基本原理
+
+在使用 INSERT INTO VALUES 时，需要通过 MySQL 协议发起导入作业给 FE 节点，FE 会生成执行计划，执行计划中前部是查询相关的算子，最后一个是 OlapTableSink 算子，用于将查询结果写到目标表中。执行计划会被发送给 BE 节点执行，Doris 会选定一个节点做为 Coordinator 节点，Coordinator 节点负责接受数据并分发数据到其他节点上。
+
+## 快速上手
+
+INSERT INTO VALUES 通过 MySQL 协议提交和传输。下例以 MySQL 命令行为例，演示通过 INSERT INTO VALUES 提交导入作业。
+
+详细语法可以参见 [INSERT INTO](../../../sql-manual/sql-statements/Data-Manipulation-Statements/Manipulation/INSERT.md)。
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data-operate/import/import-way/insert-into-values-manual.md:45]: New path : ../../../sql-manual/sql-statements/data-modification/DML/INSERT.md

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data-operate/import/import-way/insert-into-values-manual.md [hunk 7]
```diff
@@ -0,0 +1,309 @@
...
+
+```sql
+mysql> INSERT INTO test_table (user_id, name, age) VALUES (1, "Emily", 25), (2, "Benjamin", 35), (3, "Olivia", 28), (NULL, "Alexander", 60), (5, "Ava", 17);
+Query OK, 4 rows affected, 1 warning (0.04 sec)
+{'label':'label_a8d99ae931194d2b_93357aac59981a18', 'status':'VISIBLE', 'txnId':'61068'}
+```
+
+其中 `Query OK` 表示执行成功。`4 rows affected` 表示总共有 4 行数据被导入。`1 warnings` 表示被过滤了 1 行。
+
+当需要查看被过滤的行时，用户可以通过 [SHOW LOAD](../../../sql-manual/sql-statements/Show-Statements/SHOW-LOAD)语句。返回结果中的 URL 可以用于查询错误的数据，具体见后面 查看错误行 小结。
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data-operate/import/import-way/insert-into-values-manual.md:206]: New path : ../../../sql-manual/sql-statements/data-modification/load-and-export/SHOW-LOAD.md

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data-operate/import/import-way/insert-into-values-manual.md [hunk 8]
```diff
@@ -0,0 +1,309 @@
...
+
+```sql
+mysql> INSERT INTO test_table (user_id, name, age) VALUES (1, "Emily", 25), (2, "Benjamin", 35), (3, "Olivia", 28), (4, "Alexander", 60), (5, "Ava", 17);
+Query OK, 5 rows affected (0.04 sec)
+{'label':'label_78bf5396d9594d4d_a8d9a914af40f73d', 'status':'COMMITTED', 'txnId':'61074'}
+```
+
+数据不可见是一个临时状态，这批数据最终是一定可见的
+
+可以通过 [SHOW TRANSACTION](../../../sql-manual/sql-statements/Show-Statements/SHOW-TRANSACTION) 语句查看这批数据的可见状态。
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data-operate/import/import-way/insert-into-values-manual.md:243]: new path : ../../../sql-manual/sql-statements/transaction/SHOW-TRANSACTION.md

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data-operate/import/import-way/insert-into-values-manual.md [hunk 9]
```diff
@@ -0,0 +1,309 @@
...
+1 row in set (0.00 sec)
+```
+
+常见的错误的原因有：源数据列长度超过目的数据列长度、列类型不匹配、分区不匹配、列顺序不匹配等。
+
+可以通过环境变量 `enable_insert_strict`来控制 INSERT INTO 是否忽略错误行。
+
+## 更多帮助
+
+关于 Insert Into 使用的更多详细语法，请参阅 [INSERT INTO](../../../sql-manual/sql-statements/Data-Manipulation-Statements/Manipulation/INSERT) 命令手册，也可以在 MySQL 客户端命令行下输入 `HELP INSERT` 获取更多帮助信息。
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data-operate/import/import-way/insert-into-values-manual.md:309]: New path : ../../../sql-manual/sql-statements/data-modification/DML/INSERT.md

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-2.1/data-operate/import/import-way/insert-into-values-manual.md [hunk 10]
```diff
@@ -0,0 +1,309 @@
...
+
+## 基本原理
+
+在使用 INSERT INTO VALUES 时，需要通过 MySQL 协议发起导入作业给 FE 节点，FE 会生成执行计划，执行计划中前部是查询相关的算子，最后一个是 OlapTableSink 算子，用于将查询结果写到目标表中。执行计划会被发送给 BE 节点执行，Doris 会选定一个节点做为 Coordinator 节点，Coordinator 节点负责接受数据并分发数据到其他节点上。
+
+## 快速上手
+
+INSERT INTO VALUES 通过 MySQL 协议提交和传输。下例以 MySQL 命令行为例，演示通过 INSERT INTO VALUES 提交导入作业。
+
+详细语法可以参见 [INSERT INTO](../../../sql-manual/sql-statements/Data-Manipulation-Statements/Manipulation/INSERT.md)。
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-2.1/data-operate/import/import-way/insert-into-values-manual.md:45]: New path : ../../../sql-manual/sql-statements/data-modification/DML/INSERT.md

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-2.1/data-operate/import/import-way/insert-into-values-manual.md [hunk 11]
```diff
@@ -0,0 +1,309 @@
...
+
+### 前置检查
+
+INSERT INTO VALUES 需要对目标表的 INSERT 权限。如果没有 INSERT 权限，可以通过 [GRANT](../../../sql-manual/sql-statements/Account-Management-Statements/GRANT) 命令给用户授权。
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-2.1/data-operate/import/import-way/insert-into-values-manual.md:49]: New path : ../../../sql-manual/sql-statements/account-management/GRANT-TO.md

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-2.1/data-operate/import/import-way/insert-into-values-manual.md [hunk 12]
```diff
@@ -0,0 +1,309 @@
...
+
+```sql
+mysql> INSERT INTO test_table (user_id, name, age) VALUES (1, "Emily", 25), (2, "Benjamin", 35), (3, "Olivia", 28), (NULL, "Alexander", 60), (5, "Ava", 17);
+Query OK, 4 rows affected, 1 warning (0.04 sec)
+{'label':'label_a8d99ae931194d2b_93357aac59981a18', 'status':'VISIBLE', 'txnId':'61068'}
+```
+
+其中 `Query OK` 表示执行成功。`4 rows affected` 表示总共有 4 行数据被导入。`1 warnings` 表示被过滤了 1 行。
+
+当需要查看被过滤的行时，用户可以通过 [SHOW LOAD](../../../sql-manual/sql-statements/Show-Statements/SHOW-LOAD)语句。返回结果中的 URL 可以用于查询错误的数据，具体见后面 查看错误行 小结。
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-2.1/data-operate/import/import-way/insert-into-values-manual.md:206]: New path : ../../../sql-manual/sql-statements/data-modification/load-and-export/SHOW-LOAD.md

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-2.1/data-operate/import/import-way/insert-into-values-manual.md [hunk 13]
```diff
@@ -0,0 +1,309 @@
...
+
+```sql
+mysql> INSERT INTO test_table (user_id, name, age) VALUES (1, "Emily", 25), (2, "Benjamin", 35), (3, "Olivia", 28), (4, "Alexander", 60), (5, "Ava", 17);
+Query OK, 5 rows affected (0.04 sec)
+{'label':'label_78bf5396d9594d4d_a8d9a914af40f73d', 'status':'COMMITTED', 'txnId':'61074'}
+```
+
+数据不可见是一个临时状态，这批数据最终是一定可见的
+
+可以通过 [SHOW TRANSACTION](../../../sql-manual/sql-statements/Show-Statements/SHOW-TRANSACTION) 语句查看这批数据的可见状态。
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-2.1/data-operate/import/import-way/insert-into-values-manual.md:243]: same as above

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-2.1/data-operate/import/import-way/insert-into-values-manual.md [hunk 14]
```diff
@@ -0,0 +1,309 @@
...
+1 row in set (0.00 sec)
+```
+
+常见的错误的原因有：源数据列长度超过目的数据列长度、列类型不匹配、分区不匹配、列顺序不匹配等。
+
+可以通过环境变量 `enable_insert_strict`来控制 INSERT INTO 是否忽略错误行。
+
+## 更多帮助
+
+关于 Insert Into 使用的更多详细语法，请参阅 [INSERT INTO](../../../sql-manual/sql-statements/Data-Manipulation-Statements/Manipulation/INSERT) 命令手册，也可以在 MySQL 客户端命令行下输入 `HELP INSERT` 获取更多帮助信息。
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-2.1/data-operate/import/import-way/insert-into-values-manual.md:309]: same as above

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-3.0/data-operate/import/import-way/insert-into-values-manual.md [hunk 15]
```diff
@@ -0,0 +1,309 @@
...
+
+## 快速上手
+
+INSERT INTO VALUES 通过 MySQL 协议提交和传输。下例以 MySQL 命令行为例，演示通过 INSERT INTO VALUES 提交导入作业。
+
+详细语法可以参见 [INSERT INTO](../../../sql-manual/sql-statements/Data-Manipulation-Statements/Manipulation/INSERT.md)。
+
+### 前置检查
+
+INSERT INTO VALUES 需要对目标表的 INSERT 权限。如果没有 INSERT 权限，可以通过 [GRANT](../../../sql-manual/sql-statements/Account-Management-Statements/GRANT) 命令给用户授权。
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-3.0/data-operate/import/import-way/insert-into-values-manual.md:49]: New path: same as above

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-3.0/data-operate/import/import-way/insert-into-values-manual.md [hunk 16]
```diff
@@ -0,0 +1,309 @@
...
+
+```sql
+mysql> INSERT INTO test_table (user_id, name, age) VALUES (1, "Emily", 25), (2, "Benjamin", 35), (3, "Olivia", 28), (NULL, "Alexander", 60), (5, "Ava", 17);
+Query OK, 4 rows affected, 1 warning (0.04 sec)
+{'label':'label_a8d99ae931194d2b_93357aac59981a18', 'status':'VISIBLE', 'txnId':'61068'}
+```
+
+其中 `Query OK` 表示执行成功。`4 rows affected` 表示总共有 4 行数据被导入。`1 warnings` 表示被过滤了 1 行。
+
+当需要查看被过滤的行时，用户可以通过 [SHOW LOAD](../../../sql-manual/sql-statements/Show-Statements/SHOW-LOAD)语句。返回结果中的 URL 可以用于查询错误的数据，具体见后面 查看错误行 小结。
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-3.0/data-operate/import/import-way/insert-into-values-manual.md:206]: New path

File: versioned_docs/version-2.1/data-operate/import/import-way/insert-into-values-manual.md [hunk 17]
```diff
@@ -0,0 +1,305 @@
...
+
+```sql
+mysql> INSERT INTO test_table (user_id, name, age) VALUES (1, "Emily", 25), (2, "Benjamin", 35), (3, "Olivia", 28), (NULL, "Alexander", 60), (5, "Ava", 17);
+Query OK, 4 rows affected, 1 warning (0.04 sec)
+{'label':'label_a8d99ae931194d2b_93357aac59981a18', 'status':'VISIBLE', 'txnId':'61068'}
+```
+
+`Query OK` indicates successful execution. `4 rows affected` indicates that a total of 4 rows of data have been imported. `1 warnings` indicates the number of rows that were filtered out. 
+
+You can use the [SHOW LOAD](../../../sql-manual/sql-statements/Show-Statements/SHOW-LOAD/) statement to view the filtered rows.
```
Comment (KassieZ) [inline versioned_docs/version-2.1/data-operate/import/import-way/insert-into-values-manual.md:199]: New path

File: versioned_docs/version-2.1/data-operate/import/import-way/insert-into-values-manual.md [hunk 18]
```diff
@@ -0,0 +1,305 @@
...
+1 row in set (0.00 sec)
+```
+
+Common reasons for errors include: source data column length exceeding destination column length, column type mismatch, partition mismatch, and column order mismatch.
+
+You can control whether INSERT INTO ignores error rows by configuring the environment variable `enable_insert_strict`.
+
+## More help
+
+For more detailed syntax on INSERT INTO, refer to the [INSERT INTO](../../../sql-manual/sql-statements/Data-Manipulation-Statements/Manipulation/INSERT/) command manual. You can also type `HELP INSERT` at the MySQL client command line for further information.
```
Comment (KassieZ) [inline versioned_docs/version-2.1/data-operate/import/import-way/insert-into-values-manual.md:305]: New path：

File: versioned_docs/version-3.0/data-operate/import/import-way/insert-into-values-manual.md [hunk 19]
```diff
@@ -0,0 +1,305 @@
...
+
+```sql
+mysql> INSERT INTO test_table (user_id, name, age) VALUES (1, "Emily", 25), (2, "Benjamin", 35), (3, "Olivia", 28), (NULL, "Alexander", 60), (5, "Ava", 17);
+Query OK, 4 rows affected, 1 warning (0.04 sec)
+{'label':'label_a8d99ae931194d2b_93357aac59981a18', 'status':'VISIBLE', 'txnId':'61068'}
+```
+
+`Query OK` indicates successful execution. `4 rows affected` indicates that a total of 4 rows of data have been imported. `1 warnings` indicates the number of rows that were filtered out. 
+
+You can use the [SHOW LOAD](../../../sql-manual/sql-statements/Show-Statements/SHOW-LOAD/) statement to view the filtered rows.
```
Comment (KassieZ) [inline versioned_docs/version-3.0/data-operate/import/import-way/insert-into-values-manual.md:199]: ../../../sql-manual/sql-statements/data-modification/load-and-export/SHOW-LOAD.md

File: versioned_docs/version-3.0/data-operate/import/import-way/insert-into-values-manual.md [hunk 20]
```diff
@@ -0,0 +1,305 @@
...
+1 row in set (0.00 sec)
+```
+
+Common reasons for errors include: source data column length exceeding destination column length, column type mismatch, partition mismatch, and column order mismatch.
+
+You can control whether INSERT INTO ignores error rows by configuring the environment variable `enable_insert_strict`.
+
+## More help
+
+For more detailed syntax on INSERT INTO, refer to the [INSERT INTO](../../../sql-manual/sql-statements/Data-Manipulation-Statements/Manipulation/INSERT/) command manual. You can also type `HELP INSERT` at the MySQL client command line for further information.
```
Comment (KassieZ) [inline versioned_docs/version-3.0/data-operate/import/import-way/insert-into-values-manual.md:305]: New path

File: docs/data-operate/import/import-way/insert-into-values-manual.md (hunk 4 above)
Comment (kaijchen) [inline docs/data-operate/import/import-way/insert-into-values-manual.md:238]:
```suggestion
You can check the visibility status of a batch of data using the [SHOW TRANSACTION](../../../sql-manual/sql-statements/transaction/SHOW-TRANSACTION.md) statement.
```

File: docs/data-operate/import/import-way/insert-into-values-manual.md (hunk 3 above)
Comment (kaijchen) [inline docs/data-operate/import/import-way/insert-into-values-manual.md:199]:
```suggestion
You can use the [SHOW LOAD](../../../sql-statements/data-modification/load-and-export/SHOW-LOAD.md) statement to view the filtered rows.
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data-operate/import/import-way/insert-into-values-manual.md (hunk 9 above)
Comment (kaijchen) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data-operate/import/import-way/insert-into-values-manual.md:309]:
```suggestion
关于 Insert Into 使用的更多详细语法，请参阅 [INSERT INTO](../../../sql-manual/sql-statements/data-modification/DML/INSERT.md) 命令手册，也可以在 MySQL 客户端命令行下输入 `HELP INSERT` 获取更多帮助信息。
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-2.1/data-operate/import/import-way/insert-into-values-manual.md (hunk 10 above)
Comment (kaijchen) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-2.1/data-operate/import/import-way/insert-into-values-manual.md:45]:
```suggestion
详细语法可以参见 [INSERT INTO](../../../sql-manual/sql-statements/data-modification/DML/INSERT.md)。
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-2.1/data-operate/import/import-way/insert-into-values-manual.md (hunk 11 above)
Comment (kaijchen) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-2.1/data-operate/import/import-way/insert-into-values-manual.md:49]:
```suggestion
INSERT INTO VALUES 需要对目标表的 INSERT 权限。如果没有 INSERT 权限，可以通过 [GRANT](../../../sql-manual/sql-statements/account-management/GRANT-TO.md) 命令给用户授权。
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-2.1/data-operate/import/import-way/insert-into-values-manual.md (hunk 12 above)
Comment (kaijchen) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-2.1/data-operate/import/import-way/insert-into-values-manual.md:206]:
```suggestion
当需要查看被过滤的行时，用户可以通过 [SHOW LOAD](../../../sql-manual/sql-statements/data-modification/load-and-export/SHOW-LOAD.md)语句。返回结果中的 URL 可以用于查询错误的数据，具体见后面 查看错误行 小结。
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-2.1/data-operate/import/import-way/insert-into-values-manual.md (hunk 13 above)
Comment (kaijchen) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-2.1/data-operate/import/import-way/insert-into-values-manual.md:243]:
```suggestion
可以通过 [SHOW TRANSACTION](../../../sql-manual/sql-statements/transaction/SHOW-TRANSACTION.md) 语句查看这批数据的可见状态。
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-3.0/data-operate/import/import-way/insert-into-values-manual.md (hunk 1 above)
Comment (kaijchen) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-3.0/data-operate/import/import-way/insert-into-values-manual.md:45]:
```suggestion
详细语法可以参见 [INSERT INTO](../../../sql-manual/sql-statements/data-modification/DML/INSERT.md)。
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-2.1/data-operate/import/import-way/insert-into-values-manual.md (hunk 14 above)
Comment (kaijchen) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-2.1/data-operate/import/import-way/insert-into-values-manual.md:309]:
```suggestion
关于 Insert Into 使用的更多详细语法，请参阅 [INSERT INTO](../../../sql-manual/sql-statements/data-modification/DML/INSERT.md) 命令手册，也可以在 MySQL 客户端命令行下输入 `HELP INSERT` 获取更多帮助信息。
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-3.0/data-operate/import/import-way/insert-into-values-manual.md (hunk 2 above)
Comment (kaijchen) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-3.0/data-operate/import/import-way/insert-into-values-manual.md:309]:
```suggestion
关于 Insert Into 使用的更多详细语法，请参阅 [INSERT INTO](../../../sql-manual/sql-statements/data-modification/DML/INSERT.md) 命令手册，也可以在 MySQL 客户端命令行下输入 `HELP INSERT` 获取更多帮助信息。
```

File: versioned_docs/version-2.1/data-operate/import/import-way/insert-into-values-manual.md (hunk 18 above)
Comment (kaijchen) [inline versioned_docs/version-2.1/data-operate/import/import-way/insert-into-values-manual.md:305]:
```suggestion
For more detailed syntax on INSERT INTO, refer to the [INSERT INTO](../../../sql-manual/sql-statements/data-modification/DML/INSERT.md) command manual. You can also type `HELP INSERT` at the MySQL client command line for further information.
```

File: versioned_docs/version-3.0/data-operate/import/import-way/insert-into-values-manual.md (hunk 20 above)
Comment (kaijchen) [inline versioned_docs/version-3.0/data-operate/import/import-way/insert-into-values-manual.md:305]:
```suggestion
For more detailed syntax on INSERT INTO, refer to the [INSERT INTO](../../../sql-manual/sql-statements/data-modification/DML/INSERT.md) command manual. You can also type `HELP INSERT` at the MySQL client command line for further information.
```

File: versioned_docs/version-3.0/data-operate/import/import-way/insert-into-values-manual.md (hunk 19 above)
Comment (kaijchen) [inline versioned_docs/version-3.0/data-operate/import/import-way/insert-into-values-manual.md:199]:
```suggestion
You can use the [SHOW LOAD](../../../sql-manual/sql-statements/data-modification/load-and-export/SHOW-LOAD.md) statement to view the filtered rows.
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-3.0/data-operate/import/import-way/insert-into-values-manual.md (hunk 16 above)
Comment (kaijchen) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-3.0/data-operate/import/import-way/insert-into-values-manual.md:206]:
```suggestion
当需要查看被过滤的行时，用户可以通过 [SHOW LOAD](../../../sql-manual/sql-statements/data-modification/load-and-export/SHOW-LOAD.md)语句。返回结果中的 URL 可以用于查询错误的数据，具体见后面 查看错误行 小结。
```

File: versioned_docs/version-2.1/data-operate/import/import-way/insert-into-values-manual.md (hunk 17 above)
Comment (kaijchen) [inline versioned_docs/version-2.1/data-operate/import/import-way/insert-into-values-manual.md:199]:
```suggestion
You can use the [SHOW LOAD](../../../sql-manual/sql-statements/data-modification/load-and-export/SHOW-LOAD.md) statement to view the filtered rows.
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-3.0/data-operate/import/import-way/insert-into-values-manual.md (hunk 15 above)
Comment (kaijchen) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-3.0/data-operate/import/import-way/insert-into-values-manual.md:49]:
```suggestion
INSERT INTO VALUES 需要对目标表的 INSERT 权限。如果没有 INSERT 权限，可以通过 [GRANT](../../../sql-manual/sql-statements/account-management/GRANT-TO.md) 命令给用户授权。
```

**Pull request merged**